src/tests/cmocka/test_utils.c

```diff
@@ -877,6 +877,37 @@ static void test_get_next_domain_flags(void **state)
 
     dom = get_next_domain(dom, gnd_flags);
     assert_null(dom);
+
+    /* Descend only to subdomains */
+    gnd_flags = SSS_GND_SUBDOMAINS | SSS_GND_INCLUDE_DISABLED;
```

Comment:
Why not `SSS_GND_ALL_SUBDOMAINS`?

Reply:
I think being explicit here helps better to understand what is going on.

Reply:
Ok